header.php

```diff
@@ -98,7 +98,8 @@
 		</div>
 		<?php if ( ! is_front_page() && pb_get_first_post_id() ) { ?>
 			<div class="reading-header">
-				<nav class="reading-header__inside">
+				<nav aria-labelledby="book-toc" class="reading-header__inside">
+					<p id="book-toc" class="screen-reader-text">Book Contents Navigation</p>
```

Comment:
Adds correct label to ToC nav element for reading interface

Reply:
You're right